modules/gapi/test/gapi_frame_tests.cpp

```diff
@@ -49,6 +49,9 @@ class TestMediaBGR final: public cv::MediaFrame::IAdapter {
         cv::MediaFrame::View::Strides ss = { m_mat.step, 0u, 0u, 0u };
         return cv::MediaFrame::View(std::move(pp), std::move(ss), Cb{m_cb});
     }
+    cv::util::any blobParams() const override {
+        return std::make_pair<std::string, int>("hello", 42);
+    }
```

Comment:
Please drop these changes after you introduce a default implementation

Reply:
Done